.github/workflows/build-executable.yml

```diff
@@ -0,0 +1,129 @@
+name: Build Flow-PHP executable
+
+on:
+  pull_request:
```

Comment:
do we need to build execs at every PR? Maybe we can schedule it for daily and then for releases?

Reply:
I would leave 2-3 options (this one is for testing), merge to the main branch, release and maybe manual trigger, or just the last two.